---
layout: pubs
title: Publications
permalink: /docs/publications/
---

List of publications from members of our group with species being studied by CaGe.  

#### 2021

- Veglia, A.J., Milford, C.R., Schizas, N.V. (2021). Isolation and genotyping of novel T4 cyanophages associated with diverse coral reef invertebrates. Coral Reefs.  doi :10.1007/s00338-021-02056-3.
<ul>
<div class="container">
    <button type="button" class="btn btn-outline-primary btn-xs" data-toggle="collapse" data-target="#abs12">ABSTRACT</button>
        <div id="abs12" class="collapse">
            In an effort to facilitate virus isolation-based studies on coral reefs, we describe here a simple holobiont virus extraction protocol that is effective at separating and concentrating virus particles from coral reef invertebrates. We demonstrate the application of this protocol by isolating and barcoding cyanophages from invertebrate holobionts associated with coral reefs in southwest Puerto Rico. Cyanophages were also isolated and barcoded from adjacent coral reef seawater. Barcoding of cyanophage isolates was done with the cyanomyovirus DNA polymerase (g43) or major capsid protein (g23) marker genes. We then utilized cyanophage sequences from Puerto Rico, along with those published previously associated with Rhode Island and Bermuda seawater, to assess the presence of cyanophage-like sequences in reef invertebrate virome, metagenome, and transcriptome sequencing libraries. The detailed holobiont virus extraction protocol successfully separated and concentrated virus particles from the tissue of 20 different species of coral reef invertebrate. Cyanophages were isolated and barcoded from 15 of these species: three scleractinian corals, a gorgonian, a corallimorpharian, a zoantharian, two hydrozoans, four species of sponges, two tunicates and a nudibranch. In total, there were 146 cyanophages isolated and barcoded from seawater (n=46) or invertebrate tissue (n=100). The majority (69%) of cyanomyovirus sequences reported in this study were novel, sharing rather low similarity (<98 % nucleotide similarity) with publicly available sequences in the NCBI nucleotide database. Sequence library mining efforts revealed evidence of cyanophage-like sequences in 23 next-generation sequencing data sets, representing 17 species of coral reef invertebrates which included seven species of stony corals, one scyphozoan, seven species of sponges, and two species of copepods. This is the first investigation into cyanophage diversity on Puerto Rico reefs and is a relevant step in coral reef virology, ideally stimulating holobiont-associated isolation efforts to further explore virus genetic and functional diversity within invertebrate tissue on the reef.
        </div>
    <a href="https://doi.org/10.1007/s00338-021-02056-3" class="btn btn-link btn-sm" role="button" target="\_blank" rel="noopener noreferrer">
        <i class="fa fa-newspaper-o fa-lg" aria-hidden="true" style="vertical-align: middle"></i> Link to publication
    </a>
</div>
</ul>

- Hammerman NM, SM Williams, AJ Veglia, JE García-Hernández, NV Schizas, JC Lang (2021) *Cladopsammia manuelensis* sensu Cairns, 2000 (Order: Scleractinia): A new distribution record for Hispaniola and Puerto Rico. Cahiers de Biologie Marine 62: 1-10. 10.21411/CBM.A.6427875E
<ul>
<div class="container">
    <button type="button" class="btn btn-outline-primary btn-xs" data-toggle="collapse" data-target="#abs11">ABSTRACT</button>
        <div id="abs11" class="collapse">
            An unfamiliar looking azooxanthellate dendrophylliid coral, observed during shallow water (3-19 m) visual censuses since 2015 in Hispaniola and since 2019 in Puerto Rico, has been identified as <i>Cladopsammia manuelensis</i> sensu Cairns, 2000 (hereafter <i>C. manuelensis</i>). Genus assignment is based on morphological (e.g. large, widely separated and divergently oriented polyps that grow by extratentacular division, either near the base of another polyp or separated from others by a flattened coenenchyme) and genetic data (CO1 gene). We highlight the taxonomic uncertainty of the species affiliation of <i>Cladopsammia manuelensis</i> as well as the unresolved status of the closely related genera <i>Cladopsammia</i>, <i>Rhizopsammia</i> and <i>Dendrophyllia</i>. Typically found deeper (at 43-366 m) in the tropical-subtropical Atlantic Ocean and western Indian Ocean, <i>C. manuelensis</i> is also known to have increased in abundance in semi-cryptic habitats over the last decade at 4- 30 m in Curaçao and occurs in transects at 10 m in Aruba. The recent appearance of <i>C. manuelensis</i> at snorkel- SCUBA depths may represent an upward expansion of its bathymetric range and/or emergence from the cryptic habitats that abound on shallow reefs. <i>Cladopsammia manuelensis</i> superficially resembles, and genetically is relatively closely related to, three, highly invasive, semi-cryptic dendrophylliid <i>Tubastraea</i> species, and may show a similar ability to spread widely on tropical western Atlantic reefs.
        </div>
    <a href="https://doi.org/10.21411/CBM.A.6427875E" class="btn btn-link btn-sm" role="button" target="\_blank" rel="noopener noreferrer">
        <i class="fa fa-newspaper-o fa-lg" aria-hidden="true" style="vertical-align: middle"></i> Link to publication
    </a>
</div>
</ul>

#### 2019

- Veglia, A., Hammerman, N., Rivera Rosaly, C., Lucas, M., Galindo Estronza, A., Corgosinho, P., & Schizas, N. (2019). Characterizing population structure of coral-associated fauna from mesophotic and shallow habitats in the Caribbean. Journal of the Marine Biological Association of the United Kingdom, 99(3), 619-629. doi:10.1017/S0025315418000413
<ul>
<div class="container">
    <button type="button" class="btn btn-outline-primary btn-xs" data-toggle="collapse" data-target="#abs9">ABSTRACT</button>
        <div id="abs9" class="collapse">
            Symbiotic relationships are a common phenomenon among marine invertebrates, forming both obligatory and facultative dependencies with their host. Here, we investigate and compare the population structure of two crustacean species associated with both shallow and mesophotic ecosystems: an obligate symbiont barnacle (<i>Ceratoconcha domingensis</i>), of the coral <i>Agaricia lamarcki</i> and a meiobenthic, free-living harpacticoid copepod (<i>Laophontella armata</i>). Molecular analyses of the Cytochrome Oxidase Subunit I (COI) gene revealed no population structure between mesophotic and shallow barnacle populations within south-west Puerto Rico (ΦST = 0.0079, P = 0.33). The absence of population structure was expected due to the pelagic naupliar larvae of the barnacles and the connectivity patterns exhibited by the coral itself within the same region. <i>Laophontella armata</i> exhibited significant structure based on the mitochondrial COI gene between the mesophotic reef ecosystem of El Seco, Puerto Rico and mangrove sediments of Curaçao (ΦST = 0.2804, P = 0.0). The El Seco and Curaçao copepods shared three COI haplotypes despite the obligatory benthic development of harpacticoid copepods and the geographic distance between the two locations. Three other COI haplotypes from El Seco exhibited higher than expected (up to 7%) intra-species variability, potentially representing three new cryptic species of harpacticoid copepods or rare, deeply divergent lineages of <i>L. armata</i>. This result is evidence for the urgent need of a deeper investigation into the meiofauna diversity associated with mesophotic coral ecosystems (MCEs), arguably the most diverse metazoan component of MCEs.
        </div>
    <a href="https://doi.org/10.1017/S0025315418000413" class="btn btn-link btn-sm" role="button" target="\_blank" rel="noopener noreferrer">
        <i class="fa fa-newspaper-o fa-lg" aria-hidden="true" style="vertical-align: middle"></i> Link to publication
    </a>
</div>
</ul>


- García-Hernández, J. E., Hammerman, N. M., Cruz-Motta, J. J., & Schizas, N. V. (2019). Associated organisms inhabiting the calcareous sponge *Clathrina lutea* in La Parguera, Puerto Rico. Caribbean Journal of Science, 49(2-3), 239-254. 10.18475/cjos.v49i2.a12
<ul>
<div class="container">
    <button type="button" class="btn btn-outline-primary btn-xs" data-toggle="collapse" data-target="#abs7">ABSTRACT</button>
        <div id="abs7" class="collapse">
            Sponges provide an array of ecological services and benefits for Caribbean coral reefs. They function as habitats for a bewildering variety of species, however limited attention has been paid in the systematics and distribution of sponge-associated fauna in the class Calcarea or for that matter of sponges in the Caribbean. The goal of this study was to characterize infaunal assemblages from a calcareous sponge, <i>Clathrina lutea</i>, across multiple reefs from the La Parguera Natural Reserve, Puerto Rico. The associated fauna from 43 C. lutea specimens yielded a total of 2,249 associated infauna distributed in seven invertebrate phyla. Arthropoda was the most abundant phylum accounting for 62.5% of total abundance, followed by Annelida (21.0%) and Nematoda (5.5%). Limited patterns of temporal or spatial variability were surmised due to the opportunistic sampling effort afforded to this investigation from the cryptic nature of this species. A concordance between our data set and those for the class Demospongiae were observed, with the most abundant associated fauna being copepods and polychaetes. However, when compared to other Calcarea, the present study found considerably more associated fauna.
        </div>
    <a href="https://doi.org/10.18475/cjos.v49i2.a12" class="btn btn-link btn-sm" role="button" target="\_blank" rel="noopener noreferrer">
        <i class="fa fa-newspaper-o fa-lg" aria-hidden="true" style="vertical-align: middle"></i> Link to publication
    </a>
    <span class="badge badge-pill badge-secondary">Sponges</span>
</div>
</ul>

- Rivera-García, L., Rivera-Vicéns, R.E., Veglia, A.J., and Schizas, N.V. (2019). De novo transcriptome assembly of the soft octocoral *Briareum asbestinum* (digitate morphology) from southwest continental shelf of Puerto Rico. Marine Genomics. 10.1016/j.margen.2019.04.001
<ul>
<div class="container">
    <button type="button" class="btn btn-outline-primary btn-xs" data-toggle="collapse" data-target="#abs6">ABSTRACT</button>
        <div id="abs6" class="collapse">
            Octocorals have now become the most visually dominant metazoan benthic taxa of most Caribbean reefs, following the precipitous decline of scleractinian corals. Yet taxonomic issues because of their extensive phenotypic plasticity are still abound. <i>Briareum asbestinum</i> one of the iconic octocorals of the shallow Caribbean coral reefs exhibits a biform morphology, the digitate and the encrusting one. The taxonomic status of each form has not been clarified, yet. Until recently, there were few genetic resources for non-model metazoans, however, affordable high-throughput DNA sequencing has removed this hindrance. We present the first transcriptome of the digitate form of <i>Briareum asbestinum</i> from southwest Puerto Rico. We used paired-end sequencing (Illumina NextSeq 500), with a total yield of 159,754,702 raw reads. De novo assembly was performed utilizing a multi-assembler approach generating 371,554 biologically true, non-redundant transcripts. Open reading frame analysis identified 102,839 putative ORFs of which 78,607 were with annotations. BUSCO analysis indicated a total of 96.4% complete orthologous genes from the metazoan dataset. The assembly presented here serves as an important new genomic reference for the <i>Briareum</i> genus that will facilitate future population and phylogenetic studies aiming to better understand the molecular basis of phenotypic plasticity exhibited throughout the genus.
        </div>
    <a href="https://doi.org/10.1016/j.margen.2019.04.001" class="btn btn-link btn-sm" role="button" target="\_blank" rel="noopener noreferrer">
        <i class="fa fa-newspaper-o fa-lg" aria-hidden="true" style="vertical-align: middle"></i> Link to publication
    </a>
</div>
</ul>

#### 2018

- Veglia, A.J., Hammerman, N.M., Rivera-Vicéns, R.E., and Schizas, N.V. 2018. De novo transcriptome assembly of the coral *Agaricia lamarcki* (Lamarck’s sheet coral) from mesophotic depths in southwest Puerto Rico. Marine Genomics. 41:6-11. 10.1016/j.margen.2018.08.003
<ul>
<div class="container">
    <button type="button" class="btn btn-outline-primary btn-xs" data-toggle="collapse" data-target="#abs5">ABSTRACT</button>
        <div id="abs5" class="collapse">
            <p>
            The plating coral, <i>Agaricia lamarcki</i> is a widely distributed species inhabiting reefs across the Caribbean basin and Florida. This species is of interest since it is considered a depth-generalist, found from 10 to 70 m. Given the scope of contemporary studies on this coral's population dynamics and physiology, as well as, the potential of mesophotic reefs to be refuge habitats for deteriorated shallow water reefs, we present the first <i>de novo</i> transcriptome assembly of an important mesophotic coral. Using next-generation paired-end sequencing (Illumina Hiseq4000; 2 × 150 bp), we obtained a total of 82,506,058 raw reads. The novel transcriptome assembly strategy included the recently developed National Center for Genome Analysis Support <i>de novo</i> transcriptome assembly pipeline. Assembly produced a total of 101,322 biologically true, non-redundant transcripts with an average contig length of 959 and N50 of 1830. EvidentialGene and TransDecoder were used to identify open reading frames (ORFs) with homology insight provided by the UniProtKb and PFAM databases. ORF prediction resulted in 38,517 putative ORFs of which 12,107 ORFs were annotated as genes dealing with molecular function, 1266 with biological processes and 416 with cellular components.
            </p>
        </div>
    <a href="https://doi.org/10.1016/j.margen.2018.08.003" class="btn btn-link btn-sm" role="button" target="\_blank" rel="noopener noreferrer">
        <i class="fa fa-newspaper-o fa-lg" aria-hidden="true" style="vertical-align: middle"></i> Link to publication
    </a>
</div>
</ul>
- Hammerman, N. M., Rivera-Vicéns, R.E., Galaska, M. P., Weil, E., Appledoorn, R. S., Alfaro, M., & Schizas, N. V. (2018). Population connectivity of the plating coral *Agaricia lamarcki* from southwest Puerto Rico. Coral Reefs, 37(1), 183-191. 10.1007/s00338-017-1646-x
<ul>
<div class="container">
    <button type="button" class="btn btn-outline-primary btn-xs" data-toggle="collapse" data-target="#abs4">ABSTRACT</button>
        <div id="abs4" class="collapse">
            Identifying genetic connectivity and discrete population boundaries is an important objective for management of declining Caribbean reef-building corals. A double digest restriction-associated DNA sequencing protocol was utilized to generate 321 single nucleotide polymorphisms to estimate patterns of horizontal and vertical gene flow in the brooding Caribbean plate coral, <i>Agaricia lamarcki</i>. Individual colonies (n = 59) were sampled from eight locations throughout southwestern Puerto Rico from six shallow (~ 10–20 m) and two mesophotic habitats (~ 30–40 m). Descriptive summary statistics (fixation index, F ST), analysis of molecular variance, and analysis through landscape and ecological associations and discriminant analysis of principal components estimated high population connectivity with subtle subpopulation structure among all sampling localities.
        </div>
    <a href="https://doi.org/10.1007/s00338-017-1646-x" class="btn btn-link btn-sm" role="button" target="\_blank" rel="noopener noreferrer">
        <i class="fa fa-newspaper-o fa-lg" aria-hidden="true" style="vertical-align: middle"></i> Link to publication
    </a>
</div>
</ul>

#### 2017

- Ortiz-González, I.C., Rivera-Vicéns, R.E., & Schizas, N.V. (2017). De novo transcriptome assembly of the hydrocoral *Millepora alcicornis* (branching fire coral) from the Caribbean. Marine genomics, 32, 27-30. 10.1016/j.margen.2016.11.005
<ul>
<div class="container">
    <button type="button" class="btn btn-outline-primary btn-xs" data-toggle="collapse" data-target="#abs3">ABSTRACT</button>
        <div id="abs3" class="collapse">
            The hydrocoral <i>Millepora</i> is found in shallow tropical/subtropical regions around the globe and is considered an important reef-building organism. <i>Millepora alcicornis</i> is the most common species in the Atlantic Ocean, and can be found from 0.5 to 50 m deep. It is distributed from the tropical/subtropical eastern western Atlantic Ocean, including Bermuda, Brazil, and on the east in Tenerife of the Canary Islands, the Cape Verde Archipelago and Ascension Island. No genomic information is available for this ecologically important group. Here, we report de novo transcriptome assembly of <i>M. alcicornis</i> sampled from Puerto Rico, Caribbean. We used paired-end sequencing (Illumina HiSeq4000, 2 × 150 bp) and obtained 76,518,693 reads. Transcriptome assembly was performed using Trinity, producing a total of 479,982 transcripts with an average size of 553 bp and a N50 of 749. Data was normalized using RSEM and filtered by a TPM of 3. Open reading frames (ORFs) from the filtered transcripts were obtained by TransDecoder using the hydrozoan <i>Hydra vulgaris</i> protein sequences as reference, generating 16,024 putative ORFs. Blast searches showed that 25.8% (4137) of the ORFs matched <i>H. vulgaris</i> and 24.6% matched other anthozoan cnidarians (i.e. <i>Nematostella vectensis</i> = 1621, <i>Exaiptasia pallida</i> = 1280, <i>Acropora digitifera</i> = 1050). Gene ontology generated by Blast2GO resulted in a total of 7220 ORFs associated with molecular function, 4917 with biological processes, and 2989 with cellular components. A general overview will be presented of the first assembled <i>M. alcicornis</i> transcriptome with emphasis on shared genes among Hydrozoans and Scyphozoans.
        </div>
    <a href="https://doi.org/10.1016/j.margen.2016.11.005" class="btn btn-link btn-sm" role="button" target="\_blank" rel="noopener noreferrer">
        <i class="fa fa-newspaper-o fa-lg" aria-hidden="true" style="vertical-align: middle"></i> Link to publication
    </a>
</div>
</ul>

#### 2016

- Corgosinho, P. H., Schizas, N. V., & Lozano, M. A. (2016). A new species of *Atergopedia* (Copepoda: Harpacticoida: Novocriniidae) from a Caribbean mesophotic reef. Marine Biodiversity, 46(4), 841-852.
<ul>
<div class="container">
    <button type="button" class="btn btn-outline-primary btn-xs" data-toggle="collapse" data-target="#Corgosinho2016">ABSTRACT</button>
        <div id="Corgosinho2016" class="collapse">
            Coordinated research efforts in mesophotic coral reefs are only a decade old, but have already produced exciting findings in ecology, physiology, geomorphology, population genetics, biodiversity, and taxonomy. In this taxonomic contribution, we describe both sexes of a new copepod species of the family Novocriniidae Huys and Iliffe in Zool Scr 27(1):1–15, 1998 from a mesophotic coral reef off southwestern Puerto Rico. This is only the fifth species of the family and the fourth of the genus <i>Atergopedia</i> Martínez Arbizu and Moura in Zool Beitr NF 38:189–210, 1998. The new species, <i>Atergopedia dowgialli</i> sp. nov., shares the following characters with its three congeners: 1) antenna without seta on the first endopodal segment; 2) lateral armature of the female antenna shows a multiplication of distinct setae and consists of one geniculate seta followed by five or more slender setae; 3) the first endopod segment of the first swimming leg P1 bears no inner seta; 4) male fifth leg with two spines on endopodal lobe; and 5) the exopod of the fifth swimming leg fused to basendopod in the female. <i>Atergopedia dowgialli</i> sp. nov. is characterized by the loss of the basal seta in both antenna and maxilliped, the loss of the first maxilla endite, the presence of a long process between the outer and distal elements of the leg 1 exopodite 3, which is stronger and longer in the female, and the possession of a very short and strongly ornamented furca in both sexes. The genus <i>Atergopedia</i> has been found in deep and relatively shallow habitats from the Arctic Ocean to the Caribbean. In contrast, almost all species of the taxon Novocrinidimorpha, which includes the families Novocriniidae, Superornatiremidae Huys 1996, and Rotundiclipeidae Huys 1988, have been found associated with anchialine caves, indicating a derived distribution for <i>Atergopedia</i>.
>>>>>>> 15428528bc13f9738dd16d8a32f0f43ac0db59ee
        </div>
    <a href="https://doi.org/10.1007/s12526-016-0446-9" class="btn btn-link btn-sm" role="button" target="\_blank" rel="noopener noreferrer">
        <i class="fa fa-newspaper-o fa-lg" aria-hidden="true" style="vertical-align: middle"></i> Link to publication
    </a>    
</div>
</ul>

#### 2015

- Schizas, N. V., Dahms, H. U., Kangtia, P., Corgosinho, P. H., & Galindo Estronza, A. M. (2015). A new species of *Longipedia* Claus, 1863 (Copepoda: Harpacticoida: Longipediidae) from Caribbean mesophotic reefs with remarks on the phylogenetic affinities of Polyarthra. Marine Biology Research, 11(8), 789-803. 10.1080/17451000.2015.1013556
<ul>
<div class="container">
    <button type="button" class="btn btn-outline-primary btn-xs" data-toggle="collapse" data-target="#abs8">ABSTRACT</button>
        <div id="abs8" class="collapse">
            Mesophotic coral reefs are largely unexplored nearshore habitats. We present illustrations of both sexes of a new copepod species of the family Longipediidae Sars, 1903 collected via scuba-diving with tri-mix rebreathers from mesophotic coral reefs in Puerto Rico as an example of the rich fauna encountered in these habitats. The new species, *Longipedia gonzalezi*, displays a conservative morphology that characterizes the genus by having a robust body, first pedigers fused to the cephalosome, P4 exp-2 with only 1 medial seta, number of enp-1 medial setae of P1–P4 = 1:1:1:1, number of enp-2 medial setae of P1–P4 = 1:1:2:1, and caudal rami of cylindrical shape about twice as long as wide. The new species belongs to the helgolandica species-group and is morphologically similar to *L. helgolandica* and *L. americana*. Species distinction within the genus Longipedia based on morphological characters is challenging because of the conservative morphology of the genus, so we provide the first DNA sequences (28S gene) for future comparisons within the genus. The phylogenetic position of the Polyarthra (a taxon consisting of the families Longipediidae and Canuellidae, whose affinities with Harpacticoida have been questioned) is discussed within Copepoda and other Crustacea. The 28S DNA analysis confirms that Polyarthra are very closely related and are included in the strongly supported Copepoda clade. Polyarthra were not found within the Harpacticoida, which, therefore, remained not as a monophyletic, but as a paraphyletic taxon. Therefore, our 28S data indicate that the claim by Dahms and previous authors for the exclusion of Polyarthra from the Harpacticoida may be valid, and warrants further investigation.
        </div>
        <a href="https://doi.org/10.1080/17451000.2015.1013556" class="btn btn-link btn-sm" role="button" target="\_blank" rel="noopener noreferrer">
            <i class="fa fa-newspaper-o fa-lg" aria-hidden="true" style="vertical-align: middle"></i> Link to publication
        </a>    
</div>
</ul>

#### 2013

- Corgosinho, P., & Schizas, N. (2013). <i>Archeolourinia shermani</i>, a new genus and species of Louriniidae (Copepoda: Harpacticoida) from a Caribbean mesophotic zone. Journal of the Marine Biological Association of the United Kingdom, 93(3), 651-657. doi:10.1017/S0025315412001336
<ul>
<div class="container">
    <button type="button" class="btn btn-outline-primary btn-xs" data-toggle="collapse" data-target="#abs10">ABSTRACT</button>
        <div id="abs10" class="collapse">
            Mesophotic coral ecosystems (MCEs) are found on the insular and continental slopes of Caribbean islands and comprise mainly scleractinian corals, sponges and macroalgae. These species provide habitat for a highly diverse and specialized crustacean fauna. A new genus and species of the family Louriniidae is described from samples taken from an MCE in south-west Puerto Rico. The new taxa can be diagnosed by: body elongate, almost cylindrical and with inconspicuous podoplean boundary between prosome and urosome; double genital somite with a discontinuous chitinized cuticular ridge; telson short with rectangular anal operculum; furca short; rostrum sinusoidal, well defined at the base; antennules 7-segmented with aesthetasc on the fourth segment; antenna with a brush-like basal seta and an abexopodal seta, exopod 1-segmented with 2 setae; endopod with 7 elements (1 geniculate seta and 1 modified inner spine); maxilliped well developed, prehensile, composed of syncoxa, basis, one segmented endopod and one claw-like apical seta; leg 1 endopod 3-segmented, first segment with a modified inner seta; legs 2, 3 and 4 with 2-segmented endopod with inner seta on the first segment; leg 5 basendopod fused, well developed with 4 setae, exopod well developed, with 5 setae; leg 6 represented by a single seta; and genital slits wide apart. The new genus and species can be confidently assigned to the family Louriniidae on the basis of the following synapomorphies: rostrum well-developed with rounded tip; antennule of female 7-segmented; mandible palp reduced; maxillule endopod and exopod absent; maxillae endopodite 2 represented by 3 setae; leg 5 basoendopod of female confluent, intercoxal sclerite absent, exopod 1-segmented; eggs retained in a single ventral egg sac; telson shorter than last urosomite; and furca short and with 6 setae.
        </div>
        <a href="https://doi.org/10.1017/S0025315412001336" class="btn btn-link btn-sm" role="button" target="\_blank" rel="noopener noreferrer">
            <i class="fa fa-newspaper-o fa-lg" aria-hidden="true" style="vertical-align: middle"></i> Link to publication
        </a>    
</div>
</ul>
